cps/templates/index.xml

```diff
@@ -15,101 +15,101 @@
   </author>
   <entry>
     <title>{{_('Hot Books')}}</title>
-    <link href="{{url_for('opds.feed_hot')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_hot')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_hot')}}</id>
     <updated>{{ current_time }}</updated>
     <content type="text">{{_('Popular publications from this catalog based on Downloads.')}}</content>
   </entry>
   <entry>
     <title>{{_('Top Rated Books')}}</title>
-    <link href="{{url_for('opds.feed_best_rated')}}" type="application/atom+xml;profile=opds-catalog"/>
+    <link href="{{url_for('opds.feed_best_rated')}}" type="application/atom+xml;profile=opds-catalog;kind=acquisition"/>
     <id>{{url_for('opds.feed_best_rated')}}</id>
```

Comment:
Calibre adds this entry to it's feed and many opds reader are more compatible to calibre than to the spec and some of them are very strict, if there is anything different form what they are expecting it fails. Last time I changed something several readers caused problems -> No change
Example:
  <entry>
    <title>Von Neue</title>
    <id>calibre-navcatalog:9302fc328c142a43b4f37efeb050ef65e9013b98</id>
    <updated>2020-02-01T08:48:56+00:00</updated>
    <content type="text">Bücher sortiert nach Datum</content>
    <link type="application/atom+xml;type=feed;profile=opds-catalog" href="/opds/navcatalog/4f6e6577657374?library_id=Ebooks"/>
  </entry>
It differs for current behavior, yes. If you give me an example of an non working reader I'm willing to check